spec/features/request_options_spec.rb

```diff
@@ -119,55 +108,4 @@
       end
     end
   end
-
-  describe 'An In-transit discharged item', js: true do
-    before do
-      visit '/catalog/9741216'
-    end
-
-    it 'does not display a request button', unless: in_ci? do
-      using_wait_time 5 do
-        expect(page.all('.holding-block').length).to eq 1
-        expect(page).to have_selector('td[data-requestable="false"]')
-      end
-    end
-  end
-
-  describe 'Paging location available status', js: true do
-    before do
```

Comment:
Paging location could be replaced with 'Pick up' but we are checking in other tests for the 'Request' button.